tests/s3/copy_object/test_copy_obj_simultaneous_del.py

```diff
@@ -0,0 +1,221 @@
+        if os.path.exists(self.file_path):
+            os.remove(self.file_path)
+        LOGGER.info("Delete the objects and buckets created")
+        self.s3_obj.delete_multiple_buckets(self.buckets)
+        LOGGER.info("ENDED: Test teardown.")
+
+    @classmethod
+    def teardown_class(cls):
+        """
+        This is called after all tests in this class finished execution
+        """
+        LOGGER.info("STARTED: teardown test suite operations.")
+        if path_exists(cls.test_dir_path):
+            remove_dirs(cls.test_dir_path)
+        LOGGER.info("Cleanup test directory: %s", cls.test_dir_path)
+        LOGGER.info("ENDED: teardown test suite operations.")
+
+    @staticmethod
+    def copy_object_wrapper(src_bucket, src_obj, dest_bucket, dest_obj, exception=None):
+        """Copy object wrapper for multiprocessing"""
+        s3_obj = s3_test_lib.S3TestLib()
+        try:
+            return s3_obj.copy_object(src_bucket, src_obj, dest_bucket, dest_obj)
+        except CTException as err:
+            LOGGER.info("Exception in copy object %s", err)
+            if exception:
+                return exception in err.message, f"Expected {exception} Received {err}"
+            return False, f"Unexpected exception {err}"
+
+    def create_put_object(self, bucket, obj, size, base='1M'):
+        """Create and Put object of (size*1M) size and assert"""
+        resp = system_utils.create_file(fpath=self.file_path, count=size, b_size=base)
+        assert_utils.assert_true(resp[0], resp[1])
+        return self.s3_obj.put_object(bucket, obj, self.file_path)
+
+    @staticmethod
+    def delete_object_wrapper(bucket, obj):
+        """ Delete object wrapper for multiprocessing"""
+        s3_obj = s3_test_lib.S3TestLib()
+        return s3_obj.delete_object(bucket, obj)
+
+    def create_put_parallel_copy_and_delete_object(self, copy_args, delete_args, is_mpu=False):
+        """Create and uplaod object to bucket and then Parallel copy object and put object with
+        given arguments"""
+        for obj_size in [1024, 5*1024]:
+            if obj_size == 1024:
+                LOGGER.info("Upload a large object (~1GB) srcobj to srcbuck")
+            else:
+                LOGGER.info("Upload a large object (~5GB) srcobj to srcbuck")
+            if is_mpu:
+                resp = system_utils.create_file(fpath=copy_args[6], count=obj_size, b_size="1M")
+                assert_utils.assert_true(resp[0], resp[1])
+                resp = self.s3mp_obj.simple_multipart_upload(copy_args[0], copy_args[1], obj_size,
+                                                             copy_args[6], copy_args[5])
+            else:
+                self.create_put_object(copy_args[0], copy_args[1], obj_size)
+            LOGGER.info("Parallely, \n1. From %s copy %s to %s as %s\n2. Delete %s from %s",
+                        copy_args[0], copy_args[1], copy_args[2], copy_args[3], delete_args[1],
+                        delete_args[1])
```

Comment:
Line 134 : It should be delete_args[0] , can you confirm this ?

Reply:
yes, thanks